api/package.json

```
@@ -4,14 +4,15 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start:dev": "tsnd --respawn src/index.ts --dev",
+    "start:dev": "npx env-cmd -f .env.local tsnd --respawn src/index.ts --dev",
+    "start:testnet": "npx env-cmd -f .env.testnet tsnd --respawn src/index.ts --dev",
     "start": "node dist/index.js",
     "start:worker": "node dist/index.js --worker",
     "build": "tsc",
     "format": "prettier \"src/**/*.ts\" \"test/**/*.ts\"",
     "format:check": "npm run format -- -c",
     "format:write": "npm run format -- --write",
-    "migrate:make": "knex migrate:make -x ts"
+    "migrate:make": "knex migrate:make -x ts",
```

Extra `,` is making json invalid
```suggestion
    "migrate:make": "knex migrate:make -x ts"
```

---

Stupid linter.